CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ You can find its changes [documented below](#070---2021-01-01).
 
 ### Added
 
+- System fonts loaded so that SVG images render text ([#1850] by [@DrGabble])
```

Comment:
you'll also need to add the link to the PR below. :)

Reply:
@DrGabble this is ready to merge except for this missing link!

Reply:
@cmyr done!